corehq/apps/app_manager/suite_xml/post_process/resources.py

```diff
@@ -0,0 +1,61 @@
+from collections import defaultdict
+from django.db import models
+
+from corehq.apps.app_manager.exceptions import ResourceOverrideError
+from corehq.apps.app_manager.suite_xml.contributors import PostProcessor
+from corehq.apps.app_manager.suite_xml.sections.resources import FormResourceContributor
+from corehq.apps.app_manager.suite_xml.xml_models import XFormResource
+
+
+class ResourceOverride(models.Model):
+    domain = models.CharField(max_length=255, null=False)
+    app_id = models.CharField(max_length=255, null=False)
+    root_name = models.CharField(max_length=32, null=False)  # matches up with class in suite_xml.xml_models
```

Comment:
Is this just matching up with one class in that file, or the `ROOT_NAME` of all classes in that file?

Reply:
Clarified in a8bc6454892 lmk if it still doesn't make sense.